code/controllers/ProcessScheduler/core/processScheduler.dm

```diff
@@ -3,37 +3,37 @@ var/global/datum/controller/processScheduler/processScheduler
 
 /datum/controller/processScheduler
 	// Processes known by the scheduler
-	var/tmp/datum/controller/process/list/processes = new
+	var/tmp/list/datum/controller/process/processes = new
```

Comment:
List don't support typized. Make var/tmp/list/processes

Reply:
lists do support typized.

Reply:
> lists do support typized.

considering this, should I still make the changes you want me to make?

Reply:
> lists do support typized.

Try to get var type from list.

Reply:
> Try to get var type from list.

Lists do support typized. I wouldn't make that statement if I wasn't sure it was true, if you still want me to make the changes you are asking for I can make them.

Reply:
> > Try to get var type from list.
> 
> 
> 
> Lists do support typized. I wouldn't make that statement if I wasn't sure it was true, if you still want me to make the changes you are asking for I can make them. 
> 
> 

Nanako and I have been talking about this. Nanako says it is not supported.